WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2322,3 +2322,19 @@ PrefOOOExternalMsg          = margin-top:@PrefViewGutterSpace@;
 PrefOOOTimePeriodCell       = padding:@PrefSectionInternalSpacing@ 5px;
 PrefOOOMiniDropDown         = @PrefRowMiniSelectWidget@;
 PrefOOODurationLabel        = text-transform:lowercase;
+
+######################################
+#   PREFERENCES -Mail Page
+######################################
+
+PrefMailComposeRowValueWrapperWidth            = 22%
+PrefMailBlockMessageCellWidth                  = 55%
+PrefMailComposeRowSelectWidgetWidth            = 130px
```

Comment:
If it's okay for you, you ca remove this property and use `PrefRowMiniControlWidth` property declared here: https://github.com/Zimbra/zm-web-client/pull/101/files#diff-16ae02866e4f63813de9a7825a1057e3R2269